examples/ota-provider-app/ota-provider-common/OTAProviderExample.cpp

```diff
@@ -123,6 +123,24 @@ EmberAfStatus OTAProviderExample::HandleQueryImage(chip::app::CommandHandler * c
         ChipLogDetail(SoftwareUpdate, "generated URI: %s", uriBuf);
     }
 
+    // Set Status for the Query Image Response
+    switch(queryImageBehavior) {
+        case kRespondWithUpdateAvailable: {
+            queryStatus = (strlen(mOTAFilePath) ? EMBER_ZCL_OTA_QUERY_STATUS_UPDATE_AVAILABLE : EMBER_ZCL_OTA_QUERY_STATUS_NOT_AVAILABLE);
```

Comment:
Suggest adding a log statement in the case that `kRespondWithUpdateAvailable` is set but no file is provided. I could see that being unclear if the user makes a mistake.

Reply:
Done